### PR TITLE
Rhinofi restprovider using metabase

### PR DESCRIPTION
**Pull request by dmcnultyDF**

### CODE REVIEW

File: group-generators/generators/rhinofi-power-users/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { formatUsers } from ".";
```
Comment (MartinGbz) [inline group-generators/generators/rhinofi-power-users/index.test.ts:1]: Can you remove this file?
We have a CI on GitHub that check if the group is well generated or not

File: group-generators/generators/rhinofi-power-users/index.ts [hunk 2]
```diff
@@ -1,6903 +1,53 @@
...
   GenerationFrequency,
   GroupGenerator,
 } from "topics/group-generator";
 
+type UserEntry = {
+  ethAddress: string;
+  active_weeks: number;
+};
+
+export async function formatUsers(context: GenerationContext, data: Array<UserEntry>): Promise<GroupWithData[]> {
```
Comment (MartinGbz) [inline group-generators/generators/rhinofi-power-users/index.ts:19]: There is an extra indentation on the for

File: package.json [hunk 3]
```diff
@@ -81,6 +81,7 @@
     "aws-lambda": "^1.0.7",
     "axios": "^0.27.2",
     "commander": "^9.4.0",
+    "crypto": "^1.0.1",
```
Comment (MartinGbz) [inline package.json:84]: Can you remove this package?

File: yarn.lock [hunk 4]
```diff
@@ -3693,6 +3693,11 @@ crypto-js@^4.0.0:
   resolved "https://registry.yarnpkg.com/crypto-js/-/crypto-js-4.1.1.tgz#9e485bcf03521041bd85844786b83fb7619736cf"
   integrity sha512-o2JlM7ydqd3Qk9CA0L4NL6mTzU2sdx96a+oOfPu8Mkl/PK51vSyoi8/rQ8NknZtk44vq15lmhAj9CIAGwgeWKw==
 
+crypto@^1.0.1:
```
Comment (MartinGbz) [inline yarn.lock:3696]: And remove this too

**Review by MartinGbz — COMMENTED** [review]
Hey, thanks for your group update!
I left some comments :)

File: group-generators/generators/rhinofi-power-users/index.test.ts (hunk 1 above)
Comment (dmcnultyDF) [reply group-generators/generators/rhinofi-power-users/index.test.ts]: Done.

File: group-generators/generators/rhinofi-power-users/index.ts (hunk 2 above)
Comment (dmcnultyDF) [reply group-generators/generators/rhinofi-power-users/index.ts:19]: Fixed.

File: package.json (hunk 3 above)
Comment (dmcnultyDF) [reply package.json]: Done.

File: yarn.lock (hunk 4 above)
Comment (dmcnultyDF) [reply yarn.lock]: Done.

**Review by MartinGbz — APPROVED** [review]

**Pull request merged**